index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/style.css" type="text/css" rel="stylesheet"/>
+    <!--Link to font-families-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <title>Losalini Rokocakau's Portfolio | Remote Full-Stack Developer</title>
+  </head>
+  
+  <body>
+    <!--Navbar for mobile screens-->
+    <header class="nav-mobile">
+      <img src="images/image1.jpg" alt="Geometric image in top corner" class="image-1-mobile"/>

```

Comment:
- [ ] Kindly wrap this image with an ` a ` tag for semantic purposes, since this is the logo of the page.